govcd/sample_govcd_test_config.json

```diff
@@ -22,7 +22,15 @@
 		},
 		"edgeGateway": "myedgegw",
 		"externalIp": "10.150.10.10",
-		"internalIp": "192.168.1.10"
+		"internalIp": "192.168.1.10",
+    "disk": {
+      "name": "iamnewtestdisk",
+      "size": 1024,
```

Comment:
is there a reason for the two suggested sizes to be different?

Reply:
One is for create disk test and the another one is for update disk test. 
The reason  for the two suggested sizes to be different would be if the update test just use single disk size for create and update, the size is no different in between create and update, so it cannot verify whether disk size is updated correctly.

The unit of measurement for the size is byte.